### ISSUE TITLE
Should multiple invocations of jar { jruby {} } update the configuration?

### ISSUE THREAD

**Issue by rtyler** [jruby-gradle/jruby-gradle-plugin#137]
In a [plugin I am working on](/lookout/service-artifact-gradle-plugin) I am trying to set some default behaviors, which can be tweaked if a user desires.

In practice, this means that on the plugin's `apply()` code similar to this executes:

```groovy
Task artifactTask = project.task.create('assembleArtifact', type: JRubyJar) {
  /* ...snip... */
  jruby {
    defaultMainClass()
    defaultGems()
    initScript runnable()
  }
}
```

Further on in the evaluation, I would *like* to invoke:

```groovy
/** Overwrite the default entry point for the artifact */
void mainScript(String pathToScript) {
  project.tasks.findByName('assembleArtifact')?.jruby {
    initScript pathToScript
  }
}
```

This is currently not possible as the `runnable()` behavior is what gets introduced into the jar. 

So I've got two questions, mostly for @ysb33r and I to make a decision on:

1. Should this be possible?
1. Provided it occurs in the configuration phase, is there any reason that all the `JRubyJarConfigurator` properties should not be mutable?

**Comment by mkristian** [comment]
@rtyler after my patch which is still there is no configurator anymore and
those mutable restrictions are gone (maybe not all)

**Comment by ysb33r** [comment]
Has this been resolved, by @mkristian 's change?

**Comment by mkristian** [comment]
fixed via #138
